app/modules/wifi_eventmon.c

```diff
@@ -334,6 +335,7 @@ static const LUA_REG_TYPE wifi_event_monitor_reason_map[] =
   { LSTRKEY( "HANDSHAKE_TIMEOUT" ),        LNUMVAL( REASON_HANDSHAKE_TIMEOUT ) },
   { LNILKEY, LNILVAL }
 };
+LUA_TABLE_REG_2(wifi_event_monitor_reason_map);
 #endif
 
 const LUA_REG_TYPE wifi_event_monitor_map[] =
```

Comment:
Good catch -- but actually no. It turns out that the table actually has to be defined in wifi.c as that is where the reference is. I've added a comment to the code to note this.